### PR TITLE
MGMT-1339 - BMH CRs should not be created for disabled hosts

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

**Comment by yevgeny-shnaidman** [comment]
i knew i should have sent the PR to somebody else :)

File: internal/installcfg/installcfg.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ type InstallerConfigBaremetal struct {
 func countHostsByRole(cluster *common.Cluster, role models.HostRole) int {
 	var count int
 	for _, host := range cluster.Hosts {
-		if host.Role == role {
+		if *host.Status != models.HostStatusDisabled && host.Role == role {
```
Comment (filanov) [inline internal/installcfg/installcfg.go:74]:
```suggestion
		if swag.StringValue(host.Status) != models.HostStatusDisabled && host.Role == role {
```

**Comment by romfreiman** [comment]
@yevgeny-shnaidman is the title correct? Disabled hosts should not be part of the install config at all

**Comment by yevgeny-shnaidman** [comment]
yes, this is the the part of the config that deals with hosts, so the way we can check it is to see that no BMHs for created for those hosts

**Pull request merged**